src/playerOverview/PlayerScreen.tsx

```diff
@@ -41,11 +42,14 @@ import {
     teammateColumns,
 } from './playerScreenColumnHelper';
 import { PlayerScreenChampion } from './types/PlayerScreenChampion';
+import { SEASON_0_BADGE } from '../data/rewards';
 
 export async function loader(data: { params: any }) {
     return data.params.playerId;
 }
 
+const initialSeasonSelectValue = 'all_time';

```

Comment:
can/should we make the season select options enums? maybe we can store them somewhere else since they'll be reused a lot

Reply:
I thought about it, but we end up with some weird typing issues because with: 

enum seasonName = { allTime = "all_time" } 

seasonName.allTime does not equal "all_time" (one is a string, another is an enum)

I do want to use an enum though, i'll play with a little more and see what's up

Reply:
Yeah we can do that in a future PR for sure

Reply:
I think I got it working with enum